### DISCUSSION TITLE
Cant compile 2.4.3

### DISCUSSION THREAD

**Discussion by Kutsikas** [AscEmu/AscEmu#1169] [Q&A]
Trying compile  2.4.3 but get a huge error list
![1](https://github.com/AscEmu/AscEmu/assets/118775171/de6f8d4c-ee4e-4abf-bde8-6978ce1c09b4)
![2](https://github.com/AscEmu/AscEmu/assets/118775171/5482cfd1-4a5e-4099-9e3f-e4f6deaa94f1)

**Comment by schnek** [comment]
![image](https://github.com/AscEmu/AscEmu/assets/1216225/fcd18eef-de64-46c6-8e99-2b989e4314a1)
now I checked there are no errors

![image](https://github.com/AscEmu/AscEmu/assets/1216225/6c7f577c-9f94-4314-8fc3-80d7e69a9911)
it may be worth the "Delete Cache" option. and reboot

**Reply by Kutsikas** [reply]
Still same stuff cant  compile core

**Comment by aaron02** [comment]
try to build cmake in another folder not on C:/ also try to set cmake install prefix to other drive

**Comment by schnek** [comment]
as far as I understand OpenSSL were not connected correctly.. and this helped
![image](https://github.com/AscEmu/AscEmu/assets/1216225/4b54b5b1-3984-4c45-bf72-bfc9a98e070f)
